WalletWasabi/Transactions/TransactionBuilding/SmartCoinSelector.cs

```diff
@@ -29,23 +30,50 @@ public IEnumerable<ICoin> Select(IEnumerable<ICoin> coins, IMoney target)
 				throw new InsufficientBalanceException(targetMoney, available);
 			}
 
-			// Group coins by scriptPubKey in order to treat them all as one.
-			var coinsByScriptPubKey = UnspentCoins
+			var clusters = UnspentCoins
```

Comment:
What are `UnspentCoins`? Are they the selected coins or all the unspent coins?

Reply:
`UnpentCoins` are the selected coins. 

**Note:** Remember that when you select a coin, all the coins with the same `ScriptPubKey` are selected too by the `TransactionFactory`.